### PR TITLE
boards: replace @brief with @name when necessary

### PR DESCRIPTION
**Pull request by aabadie**
Only done for the boards directory. This is related  to #6466.

### CODE REVIEW

File: boards/arduino-atmega-common/include/board.h [hunk 1]
```diff
@@ -33,15 +33,16 @@ extern "C" {
 #endif
 
 /**
- * @brief As the CPU is too slow to handle 115200 baud, we set the default
- *        baudrate to 9600 for this board
+ * @name STDIO configuration
```
Comment (miri64) [inline boards/arduino-atmega-common/include/board.h]: Maybe add a line below?

File: boards/z1/include/board.h [hunk 2]
```diff
@@ -108,7 +108,7 @@ extern "C" {
 /** @} */
 
 /**
- * @brief   Definition of the interface to the CC2420 radio
+ * @name   Definition of the interface to the CC2420 radio
```
Comment (miri64) [inline boards/z1/include/board.h]: This is no group ;-)

File: boards/airfy-beacon/include/board.h [hunk 3]
```diff
@@ -28,7 +28,7 @@
 #endif
 
 /**
- * @brief   Xtimer configuration
+ * @name   Xtimer configuration
```
Comment (miri64) [inline boards/airfy-beacon/include/board.h]: Can you keep the column (i.e. add a space) everywhere in this PR?

**Comment by miri64** [comment]
This PR creates a weird collection of doxygen warnings:

```
todo:11: warning: Unexpected character `b'
todo:11: warning: Unexpected character `o'
todo:11: warning: Unexpected character `a'
todo:11: warning: Unexpected character `r'
todo:11: warning: Unexpected character `d'
todo:11: warning: Unexpected character `s'
todo:11: warning: Unexpected character `/'
todo:11: warning: Unexpected character `c'
todo:11: warning: Unexpected character `h'
todo:11: warning: Unexpected character `r'
todo:11: warning: Unexpected character `o'
todo:11: warning: Unexpected character `n'
todo:11: warning: Unexpected character `o'
todo:11: warning: Unexpected character `s'
todo:11: warning: Unexpected character `/'
todo:11: warning: Unexpected character `i'
todo:11: warning: Unexpected character `n'
todo:11: warning: Unexpected character `c'
todo:11: warning: Unexpected character `l'
todo:11: warning: Unexpected character `u'
todo:11: warning: Unexpected character `d'
todo:11: warning: Unexpected character `e'
todo:11: warning: Unexpected character `/'
todo:13: warning: Unexpected character `b'
todo:13: warning: Unexpected character `o'
todo:13: warning: Unexpected character `a'
todo:13: warning: Unexpected character `r'
todo:13: warning: Unexpected character `d'
todo:13: warning: Unexpected character `s'
todo:13: warning: Unexpected character `/'
todo:13: warning: Unexpected character `m'
todo:13: warning: Unexpected character `s'
todo:13: warning: Unexpected character `b'
todo:13: warning: Unexpected character `-'
todo:13: warning: Unexpected character `4'
todo:13: warning: Unexpected character `3'
todo:13: warning: Unexpected character `0'
todo:13: warning: Unexpected character `h'
todo:13: warning: Unexpected character `/'
todo:13: warning: Unexpected character `i'
todo:13: warning: Unexpected character `n'
todo:13: warning: Unexpected character `c'
todo:13: warning: Unexpected character `l'
todo:13: warning: Unexpected character `u'
todo:13: warning: Unexpected character `d'
todo:13: warning: Unexpected character `e'
todo:13: warning: Unexpected character `/'
todo:16: warning: Unexpected character `b'
todo:16: warning: Unexpected character `o'
todo:16: warning: Unexpected character `a'
todo:16: warning: Unexpected character `r'
todo:16: warning: Unexpected character `d'
todo:16: warning: Unexpected character `s'
todo:16: warning: Unexpected character `/'
todo:16: warning: Unexpected character `t'
todo:16: warning: Unexpected character `e'
todo:16: warning: Unexpected character `l'
todo:16: warning: Unexpected character `o'
todo:16: warning: Unexpected character `s'
todo:16: warning: Unexpected character `b'
todo:16: warning: Unexpected character `/'
todo:16: warning: Unexpected character `i'
todo:16: warning: Unexpected character `n'
todo:16: warning: Unexpected character `c'
todo:16: warning: Unexpected character `l'
todo:16: warning: Unexpected character `u'
todo:16: warning: Unexpected character `d'
todo:16: warning: Unexpected character `e'
todo:16: warning: Unexpected character `/'
todo:19: warning: Unexpected character `b'
todo:19: warning: Unexpected character `o'
todo:19: warning: Unexpected character `a'
todo:19: warning: Unexpected character `r'
todo:19: warning: Unexpected character `d'
todo:19: warning: Unexpected character `s'
todo:19: warning: Unexpected character `/'
todo:19: warning: Unexpected character `w'
todo:19: warning: Unexpected character `s'
todo:19: warning: Unexpected character `n'
todo:19: warning: Unexpected character `4'
todo:19: warning: Unexpected character `3'
todo:19: warning: Unexpected character `0'
todo:19: warning: Unexpected character `-'
todo:19: warning: Unexpected character `c'
todo:19: warning: Unexpected character `o'
todo:19: warning: Unexpected character `m'
todo:19: warning: Unexpected character `m'
todo:19: warning: Unexpected character `o'
todo:19: warning: Unexpected character `n'
todo:19: warning: Unexpected character `/'
todo:19: warning: Unexpected character `i'
todo:19: warning: Unexpected character `n'
todo:19: warning: Unexpected character `c'
todo:19: warning: Unexpected character `l'
todo:19: warning: Unexpected character `u'
todo:19: warning: Unexpected character `d'
todo:19: warning: Unexpected character `e'
todo:19: warning: Unexpected character `/'
todo:22: warning: Unexpected character `b'
todo:22: warning: Unexpected character `o'
todo:22: warning: Unexpected character `a'
todo:22: warning: Unexpected character `r'
todo:22: warning: Unexpected character `d'
todo:22: warning: Unexpected character `s'
todo:22: warning: Unexpected character `/'
todo:22: warning: Unexpected character `z'
todo:22: warning: Unexpected character `1'
todo:22: warning: Unexpected character `/'
todo:22: warning: Unexpected character `i'
todo:22: warning: Unexpected character `n'
todo:22: warning: Unexpected character `c'
todo:22: warning: Unexpected character `l'
todo:22: warning: Unexpected character `u'
todo:22: warning: Unexpected character `d'
todo:22: warning: Unexpected character `e'
todo:22: warning: Unexpected character `/'
todo:41: warning: Unexpected character `b'
todo:41: warning: Unexpected character `o'
todo:41: warning: Unexpected character `a'
todo:41: warning: Unexpected character `r'
todo:41: warning: Unexpected character `d'
todo:41: warning: Unexpected character `s'
todo:41: warning: Unexpected character `_'
todo:41: warning: Unexpected character `w'
todo:41: warning: Unexpected character `s'
todo:41: warning: Unexpected character `n'
todo:41: warning: Unexpected character `4'
todo:41: warning: Unexpected character `3'
todo:41: warning: Unexpected character `0'
todo:41: warning: Unexpected character `-'
todo:50: warning: Unexpected character `b'
todo:50: warning: Unexpected character `o'
todo:50: warning: Unexpected character `a'
todo:50: warning: Unexpected character `r'
todo:50: warning: Unexpected character `d'
todo:50: warning: Unexpected character `s'
todo:50: warning: Unexpected character `_'
todo:50: warning: Unexpected character `w'
todo:50: warning: Unexpected character `s'
todo:50: warning: Unexpected character `n'
todo:50: warning: Unexpected character `4'
todo:50: warning: Unexpected character `3'
todo:50: warning: Unexpected character `0'
todo:50: warning: Unexpected character `-'
todo:53: warning: Unexpected character `b'
todo:53: warning: Unexpected character `o'
todo:53: warning: Unexpected character `a'
todo:53: warning: Unexpected character `r'
todo:53: warning: Unexpected character `d'
todo:53: warning: Unexpected character `s'
todo:53: warning: Unexpected character `_'
todo:53: warning: Unexpected character `w'
todo:53: warning: Unexpected character `s'
todo:53: warning: Unexpected character `n'
todo:53: warning: Unexpected character `4'
todo:53: warning: Unexpected character `3'
todo:53: warning: Unexpected character `0'
todo:53: warning: Unexpected character `-'
todo:56: warning: Unexpected character `b'
todo:56: warning: Unexpected character `o'
todo:56: warning: Unexpected character `a'
todo:56: warning: Unexpected character `r'
todo:56: warning: Unexpected character `d'
todo:56: warning: Unexpected character `s'
todo:56: warning: Unexpected character `_'
todo:56: warning: Unexpected character `w'
todo:56: warning: Unexpected character `s'
todo:56: warning: Unexpected character `n'
todo:56: warning: Unexpected character `4'
todo:56: warning: Unexpected character `3'
todo:56: warning: Unexpected character `0'
todo:56: warning: Unexpected character `-'
```

File: boards/airfy-beacon/include/board.h (hunk 3 above)
Comment (aabadie) [inline boards/airfy-beacon/include/board.h]: Is this mandatory ? because it will be a pain to change that...

**Comment by aabadie** [comment]
@miri64, I addressed your comments. I had to move up the `@todo` in its own block to fix the warning. I don't know if it's a valid fix.

File: boards/airfy-beacon/include/board.h (hunk 3 above)
Comment (miri64) [inline boards/airfy-beacon/include/board.h]: Shouldn't be 's/@brief/@name /g' ;-). It's not mandatory, but the next person trying to add some doc might be happy about fitting alignment.

File: boards/chronos/include/periph_conf.h [hunk 4]
```diff
@@ -26,9 +26,16 @@ extern "C" {
...
+* @todo
+*
+* @{
+* Move all clock configuration code here from the board.h
+* @}
+*/
+/**
+ * @name   Clock configuration
  *
- * @todo    Move all clock configuration code here from the board.h
```
Comment (miri64) [inline boards/chronos/include/periph_conf.h]: Can you try moving it below the `@{`?

Comment (miri64) [inline boards/chronos/include/periph_conf.h]: (but keep as it was before)

Comment (miri64) [inline boards/chronos/include/periph_conf.h]: The problem is that `@name` names the group, but TODOs for the group need to be in the group ;-)

Comment (aabadie) [inline boards/chronos/include/periph_conf.h]: I tried several combinations, including the ones you propose, but none worked. That's why I moved the todo outside the name group

File: boards/airfy-beacon/include/board.h (hunk 3 above)
Comment (aabadie) [inline boards/airfy-beacon/include/board.h]: Unfortunately, it's not that simple since this rule can change other valid `@brief`

Comment (miri64) [inline boards/airfy-beacon/include/board.h]: Fine

File: boards/chronos/include/periph_conf.h (hunk 4 above)
Comment (miri64) [inline boards/chronos/include/periph_conf.h]: But how you do it now isn't right either. The TODO does not show up in the doc.

Comment (miri64) [inline boards/chronos/include/periph_conf.h]: I think the TODO needs to be put to a *documented* variable. I'm trying something out.

Comment (aabadie) [inline boards/chronos/include/periph_conf.h]: I tested adding /** @todo a todo line */ in between the `@name` block and the define. It works but the todo is then applied the define only not the while name block. What would you suggest ?

Comment (aabadie) [inline boards/chronos/include/periph_conf.h]: Thinking of it, maybe the best solution is to apply what's in the todo ;)

Comment (miri64) [inline boards/chronos/include/periph_conf.h]: That's what I tested too, but failed to comment due to the traps and failings of the GitHub review system ^^. I would prefer it this way. It seems like groups can't have `@todo` blocks without having this weird warnings. :-/

Comment (aabadie) [inline boards/chronos/include/periph_conf.h]: Ok, let's go for it.

Comment (aabadie) [inline boards/chronos/include/periph_conf.h]: done

**Comment by aabadie** [comment]
ping @miri64, what is your opinion on this one ?

File: boards/pic32-wifire/include/periph_conf.h [hunk 5]
```diff
@@ -33,15 +33,15 @@ extern "C" {
...
+ * @name   Timer definitions
  * @{
  */
 #define TIMER_NUMOF         (1)
 #define TIMER_0_CHANNELS    (3)
 /** @} */
 
 /**
-  * @brief   UART Definitions
+  * @name   UART Definitions
```
Comment (miri64) [inline boards/pic32-wifire/include/periph_conf.h]: At least in cases like this proper alignment would be appreciated. This just looks ugly ;-).

File: boards/nucleo144-common/include/board_common.h [hunk 6]
```diff
@@ -54,9 +54,11 @@ extern "C" {
 /** @} */
 
 /**
- * @brief   User button
+ * @name   User button
+ * @{
```
Comment (miri64) [inline boards/nucleo144-common/include/board_common.h]: Are you sure this was meant to be a group?

File: boards/iotlab-common/include/board_common.h [hunk 7]
```diff
@@ -52,16 +52,18 @@ extern "C" {
  * @name Define the interface to the AT86RF231 radio
```
Comment (miri64) [inline boards/iotlab-common/include/board_common.h]: This looks more like a variable documentation to me than a group (above too).

File: boards/arduino-atmega-common/include/board.h (hunk 1 above)
Comment (aabadie) [inline boards/arduino-atmega-common/include/board.h]: done

File: boards/fox/include/board.h [hunk 8]
```diff
@@ -34,21 +34,27 @@ extern "C" {
...
+ * @name Xtimer configuration
+ *
+ * Tell the xtimer that we use a 16-bit peripheral timer
+ * @{
  */
 #define XTIMER_WIDTH        (16)
+/** @} */
 
 /**
  * @name Define the interface to the AT86RF231 radio
```
Comment (miri64) [inline boards/fox/include/board.h]: Again: this looks more like a `@brief`, to me than a name.

File: boards/iotlab-common/include/board_common.h (hunk 7 above)
Comment (miri64) [inline boards/iotlab-common/include/board_common.h]: Dito

File: boards/nucleo144-common/include/board_common.h (hunk 6 above)
Comment (aabadie) [inline boards/nucleo144-common/include/board_common.h]: indeed I reverted it. In other places the button is sometimes defined as a group.

File: boards/iotlab-common/include/board_common.h (hunk 7 above)
Comment (aabadie) [inline boards/iotlab-common/include/board_common.h]: reverted

File: boards/pic32-wifire/include/periph_conf.h (hunk 5 above)
Comment (aabadie) [inline boards/pic32-wifire/include/periph_conf.h]: done

File: boards/airfy-beacon/include/periph_conf.h [hunk 9]
```diff
@@ -11,7 +11,7 @@
  * @{
  *
  * @file
- * @brief       Peripheral MCU configuration for the Airfy Beacon board
+ *  @brief       Peripheral MCU configuration for the Airfy Beacon board
```
Comment (miri64) [inline boards/airfy-beacon/include/periph_conf.h]: Spaceing

**Comment by aabadie** [comment]
@miri64, just pushed an update on this branch. I applied the alignment fix for every files I touched and fixed other inconsistencies (trailing underscores in header guards and unnecessary new lines).

**Comment by miri64** [comment]
Please squash ;-)

**Comment by aabadie** [comment]
squashed

**Comment by miri64** [comment]
Poor Ms. @brief and Ms. @name ^^

**Comment by aabadie** [comment]
> Poor Ms. @brief and Ms. @name ^^

indeed ;)

**Pull request merged**